src/asteroids.ts

```diff
@@ -71,13 +71,13 @@ function newAsteroid(x:number, y:number, r:number) {
  * @returns Array of Asteroids
  */
 function getRoidsInfo() {
-  return {roids: roids, roidsLeft: roidsLeft, roidsTotal: roidsTotal};
+  return { roids: roids, roidsLeft: roidsLeft, roidsTotal: roidsTotal };
```

Comment:
We can simply write
```suggestion
  return { roids, roidsLeft, roidsTotal };
```
That's how modern JS works. We can also add an ESLint rule for that. 😉

Reply:
Nice. I added the return type and it looks really clean!
```ts
function getRoidsInfo(): {
  roids: typeof roid[];
  roidsLeft: number;
  roidsTotal: number;
} {
  return { roids, roidsLeft, roidsTotal };
}
```